core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -162,4 +162,25 @@ public Cookies cookies() {
         }
         return getters.cookie();
     }
+
+    @Override
+    public RequestHeadersBuilder cookie(Cookie cookie) {
+        requireNonNull(cookie, "cookie");
+        add(HttpHeaderNames.COOKIE, cookie.toCookieHeader());
```

Comment:
- Should we use `set()`?
- Should we merge into the existing `cookie` header?

Reply:
Thank you for comment. I think we should merge it into existing header because to it's the way `AbstractHttpRequestBuilder` does. So i will fix it in that way.

Reply:
I merge it into the existing cookie header. It works fine but I think there's a better solution for that. Could you check it and give me your opinion?